client/src/components/JurisdictionAdmin/DownloadBatchTallySheetsButton/DownloadBatchTallySheetsButton.tsx

```diff
@@ -0,0 +1,81 @@
+import React, { useEffect } from 'react'
+import { AnchorButton } from '@blueprintjs/core'
+import { BlobProvider, Document } from '@react-pdf/renderer'
+import { toast } from 'react-toastify'
+
+import BatchTallySheet from './BatchTallySheet'
+import useContestsJurisdictionAdmin from '../useContestsJurisdictionAdmin'
+import { useBatches } from '../useBatchResults'
+
+const AnchorButtonErrorState = (): JSX.Element => {
+  useEffect(() => {
+    toast.error('Error preparing batch tally sheets for download')
+  }, [])
+
+  return (
+    <AnchorButton disabled icon="th">
+      Download Batch Tally Sheets
+    </AnchorButton>
+  )
+}
+
+interface IProps {
+  electionId: string
+  jurisdictionId: string
+  jurisdictionName: string
+  roundId: string
+}
+
+const DownloadBatchTallySheetsButton = ({
+  electionId,
+  jurisdictionId,
+  jurisdictionName,
+  roundId,
+}: IProps): JSX.Element | null => {
+  const batchesResp = useBatches(electionId, jurisdictionId, roundId)
+  const contests = useContestsJurisdictionAdmin(electionId, jurisdictionId)
+
+  if (!batchesResp || !batchesResp.data || !contests || contests.length === 0) {
```

Comment:
`useBatches` will always return a value. The pattern I've been starting to use for data loading using react-query looks like this:
```ts
const batchesQuery = useBatches(electionId, jurisdictionId, roundId)
if (!batchesQuery.isSuccess) return null
const { batches } = batchesQuery.data
```

It also should be impossible for `contests.length` to be 0 at this point, since the audit has launched and that requires configuring contests.

Reply:
https://github.com/votingworks/arlo/pull/1475/commits/a1aa0361379dbdbafacc01bb582b6dfaa99c0927